docs-ref-services/key-vault.md

```diff
@@ -12,141 +12,28 @@ ms.service: key-vault
 
 # Azure Key Vault libraries for Python
 
-[Azure Key Vault](/azure/key-vault/) is Azure's storage and management system for cryptographic keys, secrets, and certificate
-management. The Python SDK API for Key Vault is split between client libraries and management libraries.
+The Azure Key Vault client libraries for Python offer a convenient interface for making calls to Azure Key Vault. For more information about Azure Key Vault, see [Introduction to Azure Key Vault](https://docs.microsoft.com/en-us/azure/key-vault/general/overview).
 
-Use the following client libraries to:
-- Create, store, and control access to the keys used to encrypt your data (azure-keyvault-keys)
-- Securely store and control access to tokens, passwords, API keys, and other secrets (azure-keyvault-secrets)
-- Create, manage, and deploy SSL/TLS certificates (azure-keyvault-certificates)
+## Libraries for data access
 
-Use the management library to:
-- Create, update, or delete new Key Vault stores
-- Control vault access policies
-- List vaults by subscription or resource group
-- Check for vault name availability
+The latest version of the Azure Key Vault libraries is version 4.x. Microsoft recommends using version 4.x for new applications.
 
-## Install the libraries
+### Version 4.x
 
-### Client library
+The version 4.x client libraries for Python are part of the Azure SDK for Python. The source code for the Azure Key Vault client libraries for Python is available on [GitHub](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/keyvault).
 
-```bash
-pip install azure-keyvault-secrets 
-pip install azure-keyvault-keys
-pip install azure-keyvault-certificates
-pip install azure-identity
-```
+Use the following version 4.x libraries to work with certificates, keys, and secrets:
 
-## Examples
+| Library | Reference | Package | Source |
+|----------------------------------------|-------------------------------------------------------------|-----------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------|
+|    azure-keyvault-certificates   |    [Reference](https://azuresdkdocs.blob.core.windows.net/$web/python/azure-keyvault-certificates/4.0.0/azure.keyvault.certificates.html)    |    [Pypi](https://pypi.org/project/azure-keyvault-certificates/)    |    [GitHub](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/keyvault/azure-keyvault-certificates)    |
```

Comment:
Should check if there's a better reference doc location. We want to retire these locations, but may not have a better location for now. /cc @chlowell

Reply:
We don't have a better location yet. The preferred doc site doesn't document keyword arguments.